docs/topics/proc_deploying-a-node-js-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,38 @@
+[id='deploying-a-node-js-application-to-standalone-rhel_{context}']
+= Deploying a {Node} application to standalone Red Hat Enterprise Linux
+
+Deploy a {Node} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A {Node} application.
+* npm 6.4.1 installed
+* RHEL 7 installed.
+* {Node} installed
+
+
+.Procedure
+
+. Navigate to the directory with your application project:
++
+[source,bash,subs="attributes+",options="nowrap"]
+----
+$ cd myApp
+----
+
```

Comment:
Might want to add something about install dependecies, with `npm install`

Reply:
Thanks, added:)